docs/framework/autogen.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Using Composio With Autogen"
 sidebarTitle: "Autogen"
 icon: "robot"
-description: "Integrate Composio with Autogen agents to let them seamlessly interact with external apps"
+description: "Integrate Composio with Autogen agents to let them seamlessly interact with external Apps"
```

Comment:
Please keep 'apps' lowercase to maintain consistency with the rest of the documentation. Throughout the codebase, 'apps' is used in lowercase when referring to general applications or tools, unless specifically referring to a product name or proper noun.